src/views/partials/team/member.hbs

```diff
@@ -18,7 +18,7 @@
             <img src="{{img}}" alt="{{name}}" style="width: 100%; height: auto; max-width: 160px" />
           </div>
           <div class="col-sm-9 text-left">
-            {{description}}
+            {{{description}}}
```

Comment:
If we get rid of `<br>` we could (and probably should) store the data in plain text, not html.

Reply:
Looks like handlebars doesn't support that. We would have to use another helper function for that.